openshift/4.0/templates/maintenance/README.md

```diff
@@ -68,21 +67,49 @@ You may need to manually run the build config.
 oc start-build proxy-caddy
 ```
 
+## Deployment Configuration
+
 Create a deployment configuration file here - `deploy-proxy-caddy.dev.env`
 Update the configuration file and set the appropriate parameters.
 
 **Example**
 
 ```conf
-PROJECT_NAMESPACE=354028
-ROLE_NAME=proxy
 ENV_NAME=dev
+IMAGE_TAG=dev
 APP_DOMAIN=proxy-caddy-pims-dev.apps.silver.devops.gov.bc.ca
 APP_PORT=2015
+CPU_REQUEST=100m
+CPU_LIMIT=1
+MEMORY_REQUEST=500Mi
```

Comment:
Today I learned what a mebibyte is... lol

Reply:
me too haha